ocs_ci/ocs/resources/namespacestore.py

```diff
@@ -184,7 +184,171 @@ def verify_health(self, timeout=60, interval=5):
             ), f"{self.name} did not reach a healthy state within {timeout} seconds."
 
 
-def namespace_store_factory(request, cld_mgr, mcg_obj, cloud_uls_factory):
+def cli_create_namespacestore(
+    nss_name,
+    platform,
+    mcg_obj,
+    uls_name=None,
+    cld_mgr=None,
+    nss_tup=None,
+):
+    """
+    Create a namespace filesystem namespacestore using YAMLs
+
+    Args:
+        nss_name (str): Name of the namespacestore
+        platform (str): Platform to create the namespacestore on
+        mcg_obj (MCG): An MCG object used for executing the MCG CLI commands
+        uls_name (str): Name of the ULS bucket / PVC to use for the namespacestore
+        cld_mgr (CloudManager): CloudManager object used for supplying the needed connection credentials
+        nss_tup (tuple): A tuple containing the NSFS namespacestore details, in this order:
+            pvc_name (str): Name of the PVC that will host the namespace filesystem
+            pvc_size (int): Size in Gi of the PVC that will host the namespace filesystem
+            sub_path (str): The path to a sub directory inside the PVC FS which the NSS will use as the root directory
+            fs_backend (str): The file system backend type - CEPH_FS | GPFS | NFSv4. Defaults to None.
+
+    """
+    nss_creation_cmd = "namespacestore create "
+    NSS_MAPPING = {
+        constants.AWS_PLATFORM: lambda: (
+            f"aws-s3 {nss_name} "
+            f"--access-key {get_attr_chain(cld_mgr, 'aws_client.access_key')} "
```

Comment:
can you save `'aws_client.access_key'` as a constant? 
Same for all  the other below equivalent strings

Reply:
Why?
This can only harm readability